Scribe/Components/ParentTableViewCell/ParentTableViewCell.swift

```diff
@@ -133,6 +133,11 @@ extension ParentTableViewCell: UITableViewDelegate {
       case .appLang: break
       case .specificLang:
         if let viewController = parentViewController?.storyboard?.instantiateViewController(identifier: "TableViewTemplateViewController") as? TableViewTemplateViewController {
+          let accentKeyLanguages: [String] = ["Swedish", "German", "Spanish"]; //Languages where we can disable accent keys
+          if (accentKeyLanguages.firstIndex(of: section.sectionTitle) == nil)
+          {
+            SettingsTableData.languageSettingsData[0].section.remove(at: 1) //If there are no accent keys we remove the option.
```

Comment:
This is really great, @henrikth93! Could we add one line right before this that checks what the the index is for `Disable accent characters` and then we can drop that index specifically? As of now if something else goes in at index 1 this will break its functionality :)

Aside from this, 100% what we're looking for! 😊

Reply:
Thanks @henrikth93! Will merge it in soon after 🤠